build.gradle

```diff
@@ -6,7 +6,6 @@ buildscript {
 
     repositories {
         google()
-        mavenLocal()
```

Comment:
Doesnt need to go, it doesnt affect.

Reply:
Agree. Removed as I was refactoring the SDK and this is not needed. Is it being used on the mP side and should I include back?

Reply:
The workflow was failing so added back the mavenLocal().  Probably the mP build system is using it.